third_party/skia/src/sksl/SkSLThreadContext.cpp

```diff
@@ -15,6 +15,12 @@
 #include "src/sksl/SkSLCompiler.h"
 #include "src/sksl/ir/SkSLExternalFunction.h"
 
+#if defined(STARBOARD)
+#include "starboard/common/log.h"
+#include "starboard/once.h"
+#include "starboard/thread.h"
+#endif
+
```

Comment:
Because previously it was in the wrong namespace `SkSL`.  `#include <foo>` is never okay to put in a local namespace, it breaks things significantly.

It only worked by accident before.